### PR TITLE
Changed how React icons are generated to fix an issue with icons built in Figma

### PR DESCRIPTION
**Pull request by ericsoderberghp**
See comment in generate-icons for details.

Check the snapshot changes.

### CODE REVIEW

File: tools/generate-icons.js [hunk 1]
```diff
@@ -8,11 +8,20 @@ import xml2js from 'xml2js';
 const inputSVGFolder = path.resolve('public/img');
 const outputReactIconFolder = path.resolve('src/js/icons');
 
+// Figma converts colors to their color names. Convert them back to hex codes.
+// We do this so the rules in StyledIcon work. '#' indicates a fill or stroke
+// is desired. Otherwise, it is treated like 'none'.
+function convertColors(text) {
```
Comment (ShimiSun) [inline tools/generate-icons.js:14]: What about colors such as `red` or `blue`? should we be adding it as a need arise?

Comment (ericsoderberghp) [reply tools/generate-icons.js:14]: Add as the need arises. There are a lot of named colors. I'm not sure how Figma is behaving for them. Black and white are all we need at the moment.

**Comment by ShimiSun** [comment]
Redeploying Netlify to hopefully fix the failure.

**Comment by ShimiSun** [comment]
Background for the fix:

```
*[stroke*="#"],
  *[STROKE*="#"] {
    stroke: inherit;
    fill: none;
  }
  *[fill-rule],
  *[FILL-RULE],
  *[fill*="#"],
  *[FILL*="#"] {
    fill: inherit;
    stroke: none;
  }
```
We interpret ‘#’ as a color we should change to inherit. Otherwise, we treat it as none.

It would be complicated to match color names in those selectors.

**Pull request merged**